recipe/meta.yaml

```diff
@@ -29,8 +29,12 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - sqlite
+    - libtiff
+    - curl
```

Comment:
According to the [docs](https://www.osgeo.org/foundation-news/proj-7-0-0/), this should be `libcurl` instead of `curl`, right?